.cirrus.yml

```diff
@@ -721,14 +721,12 @@ upgrade_test_task:
     depends_on:
       - local_system_test
     matrix:
-        - env:
-              PODMAN_UPGRADE_FROM: v1.9.0
-        - env:
-              PODMAN_UPGRADE_FROM: v2.0.6
         - env:
               PODMAN_UPGRADE_FROM: v2.1.1
         - env:
               PODMAN_UPGRADE_FROM: v3.1.2
+        - env:
+              PODMAN_UPGRADE_FROM: v3.4.0
```

Comment:
quay also has v3.4.1, 2, and 4. Should we use one of those instead?

Reply:
I would go with .4 to have more bug fixes in it.